### DISCUSSION TITLE
Confused on many levels

### DISCUSSION THREAD

**Discussion by dippelmax** [sr320/course-fish546-2021#22] [General]
I was unable to work with blast, the .fa file from washington.edu and the Swiss-Prot sequences. I am not sure why these issues are occuring. I am trying to replicate the code shown in the tutorial and in the jupyter notebooks from other students, but I keep running into errors.

**Comment by kubu4** [comment]
Just documenting discussion from class today.

A couple of things. One problem was that the download directory specified in your `curl` command didn't exist. This was a common problem that other encountered. Making the desired directory structure before downloading files is helpful to avoid this. Also, using tab completion when typing the destination location can be a useful "check" to see if the specified location exists before you start downloading.

Another problem was the `curl` command for the FASTA file. For some reason, the `curl` command also needed an additional argument (`-k`) to deal with a weird server certificate issue. Totally non-intuitive, but the error message suggested this as a possible fix and it worked!
